geo/src/main/java/org/apache/carbondata/geo/GeoHashImpl.java

```diff
@@ -147,7 +206,18 @@ public void init(String handlerName, Map<String, String> properties) throws Exce
                       CarbonCommonConstants.INDEX_HANDLER, CONVERSION_RATIO));
     }
 
-    // TODO: Fill the values to the instance fields
+    // Fill the values to the instance fields
+    this.oriLatitude = Double.valueOf(originLatitude);
+    this.userDefineMaxLongitude = Double.valueOf(maxLongitude);
+    this.userDefineMaxLatitude = Double.valueOf(maxLatitude);
+    this.userDefineMinLongitude = Double.valueOf(minLongitude);
+    this.userDefineMinLatitude = Double.valueOf(minLatitude);
+    this.gridSize = Integer.parseInt(gridSize);
+    this.conversionRatio = Integer.parseInt(conversionRatio);
+    this.lon0ByRation = userDefineMinLongitude * this.conversionRatio;
+    this.lat0ByRation = userDefineMinLatitude * this.conversionRatio;
+    // calculate the data
+    calculateData();
```

Comment:
can you give better name ? what data is it ? polygon data or geo hash ID ?

Reply:
okay